src/app/Food.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package app;
+
+import javafx.scene.control.DatePicker;
+
+/**
+ *
+ * @author aswecker
+ */
+public class Food {
+    private String foodName;
+    private int servingSize;
+    private DatePicker foodDate;
+    private int id;
+    private String foodGroup;   
+    
+    /**
+    * This is the default constructor for the Food class.
+    * @param foodName
+    * @param servingSize
+    * @param foodDate
+    */     
+    public Food(int id, String foodName, int servingSize, String foodGroup, DatePicker foodDate){
+        this.id = id;
+        this.foodName = foodName;
+        this.servingSize = servingSize;
+        this.foodGroup = foodGroup;
+        this.foodDate = foodDate;
+    }
+    
+   /**
+    * Returns the id
+    * @return An integer for the id
+    */  
+    public int getId(){
+        return id;
+    }
+    
+    public void setId(int id){
+        this.id=id;
+    }
+    
+    /**
+    * Returns the foodName
+    * @return A string for the food name
+    */    
+    public String getFoodName(){
+        return foodName;
+    }
+    
+    public void setFoodName(String foodName){
+        this.foodName=foodName;
+    }
+    
+    /**
+     * Returns the servingSize
```

Comment:
We might want to put a unit in the description. We could just say grams (g) and that would be OK.